rebar.config

```diff
@@ -1,7 +1,4 @@
 {erl_opts, [debug_info]}.
-{deps, [
```

Comment:
I don't think you should modify top-level `rebar.config`, but use the one that is provided in apps/antidote/rebar.config instead. Might require some more modifications to make it work.

In general we have 4 different targets here potentially:
- antidote itself
- vaxine together with antidote bundled together
- Vax
- antidote-erlang-client
Current idea to make sure we can build all of them independently by using corresponding mix.exs/rebar.config files located accordingly.

Reply:
Because root rebar.config was intended for a vaxine node (erlang vm with vaxine layer + antidote)

Reply:
We've discussed a bit more in private and decided that it's fine for now